src/System.Data.Common/tests/System/Data/Common/DbDataReaderTest.cs

```diff
@@ -45,101 +46,103 @@ public DbDataReaderTest()
             Assert.Equal(3, testData.Rows.Count);
         }
 
-#if NET_4_5
 		[Fact]
-		public void GetFieldValueTest ()
+		public void GetFieldValueTest()
 		{
 			//First row
-			dataReader.Read ();
-			Assert.Equal ("row_1", dataReader.GetFieldValue<string> (0));
+			 _dataReader.Read();
+			Assert.Equal("row_1", _dataReader.GetFieldValue<string>(0));
 			byte[] expected_data = new byte[] { 0xde, 0xad, 0xbe, 0xef };
-			byte[] actual_data = dataReader.GetFieldValue<byte[]> (1);
-			Assert.Equal (expected_data.Length, actual_data.Length);
-			for (int i = 0; i < expected_data.Length; i++) {
-				Assert.Equal (expected_data [i], actual_data [i]);
+			byte[] actual_data = _dataReader.GetFieldValue<byte[]>(1);
+			Assert.Equal(expected_data.Length, actual_data.Length);
+			for (int i = 0; i < expected_data.Length; i++)
+			{
+				Assert.Equal(expected_data [i], actual_data [i]);
 			}
 
 			//Second row where data row column value is DBNull
-			dataReader.Read ();
-			Assert.Equal ("row_2", dataReader.GetFieldValue<string> (0));
-			try {
-				actual_data = dataReader.GetFieldValue<byte[]> (1);
-Assert.False(true);
-			} catch (InvalidCastException) {
+			 _dataReader.Read();
+			Assert.Equal("row_2", _dataReader.GetFieldValue<string>(0));
+			try
+			{
+				actual_data = _dataReader.GetFieldValue<byte[]>(1);
+                Assert.False(true);
+			}
+			catch (InvalidCastException)
```

Comment:
could be `Assert.Throws<InvalidCastException>(() => _dataReader.GetFieldValue<byte[]>(1));`